addons/website_crm/views/website_crm_templates.xml

```diff
@@ -59,10 +59,11 @@
                                             </div>
                                         </div>
                                     </div>
-                                    <div class="form-group col-12 s_website_form_field s_website_form_model_required" data-type="char" data-name="Field">
+                                    <div class="form-group col-12 s_website_form_field s_website_form_required s_website_form_model_required" data-type="char" data-name="Field">
                                         <div class="row s_col_no_resize s_col_no_bgcolor">
                                             <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="opportunity5">
                                                 <span class="s_website_form_label_content">Subject</span>
+                                                <span class="s_website_form_mark"> *</span>
```

Comment:
Note in case it might help for your forward port, there were commits in 15.0 doing similar things: https://github.com/odoo/odoo/commit/7755510dfedb2a53816e26ec68c947088b876eb2
I think there are others too